### ISSUE TITLE
when CrossEntropyLoss loss=nan

### ISSUE THREAD

**Issue by JonesonZheng** [clcarwin/sphereface_pytorch#24]
when I use 

        loss = nn.CrossEntropyLoss()(output, target) 

replace the code in AngleLoss

        logpt = F.log_softmax(output)
        logpt = logpt.gather(1,target)
        logpt = logpt.view(-1)
        pt = Variable(logpt.data.exp())
        loss = -1 * (1-pt)**self.gamma * logpt
        loss = loss.mean()
I got loss=nan

I think theoretically logpt = F.log_softmax(output) here should be logpt = F.log_softmax(output,1) which is just the same as CrossEntropyLoss, but you implement is logpt = F.log_softmax(output,0)
Would you help explain it?

**Comment by clcarwin** [comment]
You can try to compare the code in [focal_loss_pytorch](https://github.com/clcarwin/focal_loss_pytorch).

**Comment by suanrong** [comment]
@JonesonZheng

I think you are right.  It should be logpt = F.log_softmax(output,1)
